src/obr/cli.py

```diff
@@ -387,38 +394,48 @@ def archive(ctx: click.Context, **kwargs):
         )
 
     # setup target folder
-    path = Path(target_folder + f"/{time}")
-    if not path.exists():
-        logging.info(f"creating {path}")
-        path.mkdir()
+    if not target_folder.exists():
+        logging.info(f"creating {str(target_folder)}")
+        target_folder.mkdir()
 
     skip = kwargs.get("skip_logs")
     if not skip:
         # iterate cases and copy log files into target repo
         for job in jobs:
+            # copy signac state point
+            signac_statepoint = Path(job.path) / "signac_statepoint.json"
+            if not signac_statepoint.exists():
+                continue
+            else:
```

Comment:
Slight suggestion; remove indentation levels by removing the else:
Also, use logging.
 ```suggestion
        target_file = (
                    target_folder / f"workspace/{job.id}/signac_statepoint.json"
                )
        logging.debug(f"{target_folder}, {signac_statepoint}")
        copy_to_archive(repo, use_github_repo, signac_statepoint, target_file)
```